examples/responsive-polyfill.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Responsive Media Chrome Example</title>
+    <script type="module" src="../dist/index.js"></script>
+    <style>
+      /** add styles to prevent CLS (Cumulative Layout Shift) */
+      media-controller:not([audio]) {
+        display: block;         /* expands the container if preload=none */
+        max-width: 1280px;       /* allows the container to shrink if small */
+        aspect-ratio: 16 / 9;   /* set container aspect ratio if preload=none */
+
+        container: media-chrome / inline-size;
+      }
+      video {
+        width: 100%;      /* prevents video to expand beyond its container */
+      }
+
+      .centered-controls-overlay {
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: center;
+        justify-content: space-evenly;
+      }
+
+      @container (max-width: 590px) {
+        .centered-controls-overlay:not(.container-query-polyfill) {
```

Comment:
the polyfill was saying that we need to add the `:not(.container-query-polyfill)` because it's saying that `:where` isn't supported, though, supposedly, `:where` is available according to caniuse.com https://caniuse.com/mdn-css_selectors_where